src/conf.py

```diff
@@ -51,14 +52,14 @@
 html_theme = 'alabaster'
 
 html_theme_options = {
-    'logo': 'logo2.png',
+    'logo': 'logo1.png',
     'body_max_width': '75%',
     'page_width': '85%',
-    'sidebar_width': '15%'
+    'sidebar_width': '10%'
```

Comment:
This change makes sidebar squashed on my desktop browser. For example, the word 'Navigation' now spans two lines